libs/core/langchain_core/messages/modifier.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any, List, Literal
+
+from langchain_core.messages.base import BaseMessage
+
+
+class RemoveMessage(BaseMessage):
+    """Message responsible for deleting other messages."""
+
+    type: Literal["remove"] = "remove"
+
+    def __init__(self, id: str, **kwargs: Any) -> None:
+        return super().__init__("", id=id)
```

Comment:
probably worth plumbing through kwargs (with check that content isn't one of them)? seems a little more future proof